targets/TARGET_NORDIC/TARGET_NRF5x/pinmap_ex.c

```diff
@@ -0,0 +1,248 @@
+/* mbed Microcontroller Library
+ * Copyright (c) 2018 ARM Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "pinmap_ex.h"
+
+#include "mbed_toolchain.h"
+#include "nrf.h"
+#include "nrf_peripherals.h"
+
+#include <stdio.h>
+
+#if 0
```

Comment:
Should this be wrapped around a #define (MBED_DEBUG, or something similar), or is this just for debugging in the feature branch?

Reply:
Its really only for debugging this specific file if someone has to go back to the future.

Reply:
Not certain about leaving it here, I would remove it .

Reply:
@marcuschangarm Ok for now, but will definitely need to be changed before coming into master.

Reply:
Is there a requirement for this? I've noticed similar logging/debug code in partner's SDKs.

Reply:
I feel like there should be, but I'm not certain there is.
@0xc0170 @adbridge ?

Reply:
We have no control over what partners put in their SDKs, however that doesn't mean we should do such things ourselves...